x/rollapp/types/chain_id.go

```diff
@@ -66,6 +77,22 @@ func getEIP155ID(chainID string) (*big.Int, error) {
 	return chainIDInt, nil
 }
 
+// getEIP155ID parses a string chain identifier and returns the revision number
+func getRevisionNumber(chainID string) (uint64, error) {
+	matches := strings.Split(chainID, "-")
+	if len(matches) == 1 {
+		return 0, nil
+	}
+	if len(matches) != 2 {
+		return 0, errorsmod.Wrapf(ErrInvalidRollappID, "unable to parse revision number")
+	}
+	revision, err := strconv.ParseUint(matches[1], 0, 64)
+	if err != nil {
+		return 0, errorsmod.Wrapf(ErrInvalidRollappID, "unable to parse revision number: error: %e", err)
```

Comment:
%e is wrong, it should be %w or %v I think

Reply:
rplaced for %v